src/views/swap/steps/sendTransaction.js

```diff
@@ -24,39 +26,47 @@ const SendTransactionStyles = () => ({
   info: {
     flexDirection: 'column',
     flex: 1,
-    justifyContent: 'space-around',
   },
   text: {
-    fontSize: '30px',
+    fontSize: '20px',
+    '@media (max-width: 425px)': {
+      fontSize: '16px',
+    },
   },
   address: {
     width: '300px',
-    fontSize: '25px',
+    fontSize: '18px',
     color: 'grey',
     wordBreak: 'break-word',
+    '@media (max-width: 425px)': {
+      fontSize: '16px',
+    },
   },
   action: {
     color: 'blue',
     fontWeight: '600',
-    fontSize: '30px',
-    marginLeft: '50%',
+    fontSize: '20px',
+    marginLeft: '70%',
     '&:hover': {
       cursor: 'pointer',
     },
+    '@media (max-width: 425px)': {
+      fontSize: '16px',
+    },
   },
 });
 
 const StyledSendTransaction = ({ classes, swapInfo, swapResponse }) => (
   <View className={classes.wrapper}>
     <View className={classes.qrcode}>
-      <QrCode size={300} link={swapResponse.bip21} />
+      <QrCode size={250} link={swapResponse.bip21} />
     </View>
     <View className={classes.info}>
       <p className={classes.text}>
         Send{' '}
         <b>
           {' '}
-          {toWholeCoins(swapResponse.expectedAmount)} {swapInfo.base}{' '}
+          {toWholeCoins(swapResponse.expectedAmount)} {swapResponse.bip21}{' '}
```

Comment:
This should stay `swapInfo.base`
```suggestion
          {toWholeCoins(swapResponse.expectedAmount)} {swapInfo.base}{' '}
```